crucible-mir/src/Mir/PP.hs

```diff
@@ -310,6 +310,15 @@ instance Pretty BinOp where
       Unchecked op' -> pretty op' <> pretty "!"
       WithOverflow op' -> pretty op' <> pretty "?"
 
+instance Pretty VtableItem where
+  pretty (VtableItem fn def) = tupled [pretty fn, pretty def] <> semi
```

Comment:
Suggestions welcome on how to more neatly print this. I couldn't easily find reference output for how Rust vtables are conventionally displayed, so I more-or-less improvised things here.

Reply:
I'm comfortable with how this looks, though I may not be the best person to judge.

I do think the in-tuple newlines in particular are nice to have, since they make it easier to see a short difference between two elements that share a long prefix, as in your example. Will `tupled` always produce newlines like that, do you know?

Reply:
`tupled` doesn't always insert newlines, no. It attempts to lay out all the elements on one line, but it will insert a line break if it doesn't fit the width the page (usually 80 characters when using `prettyprinter`'s default settings). I could try to define this in terms of `vcat` instead to ensure that the two `DefId`s in each entry are always stacked on top of each other vertically, however.